lib/hanami/cli/server.rb

```diff
@@ -0,0 +1,45 @@
+module Hanami
+  module CLI
+    # @api private
+    class Server
+      attr_reader :rack_server
```

Comment:
@waiting-for-dev Please move this below constants. Also, please make it private `private attr_reader :rack_server`

Reply:
We're leveraging the public visibility of `:rack_server` to mock it in the test suite. Besides, I think it makes sense to mark the whole class as private for the user. The public interface is the command but not the server. WDYT?